### DISCUSSION TITLE
Unable to add the Helm Repo

### DISCUSSION THREAD

**Discussion by baburaj5** [jertel/elastalert2#1129] [Q&A] [answered]
I could not able to add helm repo running the below command.
helm repo add elastalert2 https://jertel.github.io/elastalert2/

Getting the below error, can get some help on this.
Error: looks like "https://jertel.github.io/elastalert2/" is not a valid chart repository

**Comment by jertel** [comment, marked as answer]
Worked fine for me just now. Perhaps GitHub was having issues earlier.

```
$ helm repo add elastalert2 https://jertel.github.io/elastalert2
"elastalert2" has been added to your repositories

$ helm repo list
NAME       	URL
elastalert2	https://jertel.github.io/elastalert2

$ helm search repo
NAME                   	CHART VERSION	APP VERSION	DESCRIPTION
elastalert2/elastalert2	2.10.0       	2.10.0     	Automated rule-based alerting for Elasticsearch
```
